### DISCUSSION TITLE
[Megathread] pockethost.io system status, health, and outage reports - Report Problems Here 🌮

### DISCUSSION THREAD

**Discussion by benallfree** [pockethost/pockethost#223] [Q&A]
If you experience any issues where you believe PocketHost may be down or experiencing a service disruption, please use this thread to report and discuss.

Also, head to https://discord.gg/nVTxCMEcGT for live incident chat.

**Comment by benallfree** [comment]
Ok everyone, I think there are enough of us experiencing downtime issues that we can pull together to solve it.

## Background

Several of you (including me) have noticed that PocketHost seems to intermittently disappear and then come back to life 10-20 minutes later. This has never been observed during development, so it seems to be an issue that only shows up in production and under real use conditions. 

## Possible causes and solutions

### Something about Digital Ocean

**Likelihood: extremely unlikely 🚫**

**Cause:** It is possible that the Digital Ocean network or VPS becomes inaccessible for periods of time.

**Solution:** Change providers

### A bug in in PocketBase

**Likelihood: unlikely 😒**

**Cause:**  It is possible there is an unaddressed bug in PocketBase. I consider this to be unlikely because the outages happen regardless of which version of PocketBase a user is running.

**Solution:** Identify and report bugs to the PocketBase project


### 'Bad Neighbor' theory - something about how a particular user is using PocketBase

**Likelihood: likely 🤨**

**Cause:**  We have already seen instances where a single PocketBase process will spike to max CPU usage because of a badly written or long query. This starves the other processes and has the effect of making the PocketHost server unresponsive. 

**Solution:** Isolate user processes and use [cgroups](https://man7.org/linux/man-pages/man7/cgroups.7.html) to prevent a single PocketBase process from becoming a 'bad neighbor'. This is worth implementing anyway because it would make the system more secure by disallowing system-level access to PocketBase processes.

### Something in the PocketHost code

**Likelihood: most likely 🙈**

**Cause:** Humility demands that I recognize my own code is the most likely problem :) When a web request comes in, it hits a nodejs proxy that decides what to do and routes the request accordingly. Most requests are forwarded to a secondary proxy layer (called the daemon) which is responsible for launching PocketBase instances on demand. Then, the request is forwarded to the actual PocketBase instance. We have seen cases where something in this chain of proxies stops responding. It isn't clear why, but it has been observed.

**Solution:** There is no universal answer to this. The code is not particularly deep or complicated, but it is possible we have a leak somewhere and that the process eventually runs out of some kind of resource. 

## Current action items

While I think the PocketHost code itself is the most likely culprit, I think that adding support to isolate PocketBase processes and limit resource usage per instance is a good way to create more visibility. If a PocketBase processes runs out of control due to a bad query or really for any reason at all, the nodejs code might not handle that situation well, especially if incoming requests are piling up. So I think I'll focus my energy on implementing process isolation and then see how stability changes.

**Reply by benallfree** [reply]
This is what a downtime pattern looks like.

<img width="990" alt="image" src="https://github.com/benallfree/pockethost/assets/1068356/103649ef-309b-4537-a6c9-43d53e8b4e59">

**Reply by BasToTheMax** [reply]
Why not use docker to limit the ram and cpu of an instance?

**Reply by benallfree** [reply]
@BasToTheMax Do you mean launch each pocketbase instance inside a docker container? That could work...

Originally I had all of PocketHost running in a docker container and that was decidedly NOT helpful.

**Reply by BasToTheMax** [reply]
Yes, I indeed mean to have each instance in it's own docker container

**Reply by benallfree** [reply]
@BasToTheMax Do you want to take this task on?

**Reply by BasToTheMax** [reply]
I can try, but I can't guarantee anything.

**Reply by benallfree** [reply]
@BasToTheMax You could begin by figuring out a Docker solution that:

1. Runs PocketBase inside Docker on a specific architecture that matches the host OS
2. Limits the CPU and memory across all running Docker instances. For example, we might want to limit the GROUP of docker processes to 500mb memory and 80% CPU. I don't know if Docker is capable of this type of resource pooling, but that is essentially what [cgroup](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/ch01) does.

Thank you if you find out those answers!

**Reply by cheskoxd** [reply]
If you are already deploying the instances with docker containers, what if we set some constraints for the resources?

Like this (docker-compose.yml)

```
version: '3.8'
services:
  app:
    image:pocketbase:latest
    ports:
      - "8080:80"
    environment:
      - ENV_VAR_1=something
    deploy:
      resources:
        limits:
          cpus: '0.5' # Maximum 50% of a CPU core per instance
          memory: '256M' # Maximum 256MB of memory per instance
        reservations:
          cpus: '0.1' # Minimum 10% of a CPU core per instance
          memory: '128M' # Minimum 128MB of memory per instance

```

Not sure if this helps or if it's out of topic, but as you said, restricting each instance to a max amount of memory and CPU will be better, and I think this could work?...

At the end docker uses cgroups to limit resources on instances

**Reply by cheskoxd** [reply]
Also If you are using docker swarm  you can specify a limit of resources with

```
docker service update --limit-cpu=80m --limit-memory=10G your_service_name
```

**Reply by benallfree** [reply]
Started separate discussion https://github.com/benallfree/pockethost/discussions/269

**Comment by nicosql** [comment]
@benallfree it seems like long term maybe the solution is to isolate the environments, even if it is your code - your "bad neighbor" scenario never goes away and becomes worse as your repo (and pocketbase) grow in popularity.

Have you considered building on top of something like railway.io?  

They use a Docker implementation of Pocketbase and while it is fairly simple to use that is subjective and would still require some Docker knowledge.

I can still see a platform like Pockethost still being very useful to the community given the roadmap and features you already provide.

There is a generous free tier and it would inherently isolate each instance and you could impose some hard capped limit more easily so the experience is stable.


I think what you're doing is great but the product manager in me sees this as non-viable for a single person long term and I want pockethose and pocketbase to go on!  That's my input, I will be watching with a learning eye. :)

**Reply by benallfree** [reply]
@pookiepats Do you mean https://railway.app/?

Early on, I had considered a different architecture where each user would need to input their own fly.io credentials and then PocketHost would provision resources in a completely separate environment. Something like that is still a possibility, and yes the 'shared hosting' model is bound to break down eventually.

I'd be open to getting some help with this aspect. The immediate issue would be deciding which cloud platforms to support (aws, google, fly, etc?). Then PocketHost could just become a management console and leave the hosting to the experts.

Thoughts?

**Reply by BasToTheMax** [reply]
Railway doesn't have persistent storage

**Reply by nicosql** [reply]
@benallfree haha yes, railway.app is what I meant and yes I really like that idea - for people like me (I'm about 8 months into my Javascript / web dev journey coming from years of deep database admin type roles) - platforms like this are a dream.

Yes the services make it easy enough once you're familiar but having to jack with all the minutia (.env, CI, domains, dockerfiles etc..) can really rob that learner's joy.

I'm not sure how else I can help though, got a donation link or something? lol 

I do like fly.io though, I've used them for some LiveView stuff and it was pretty slick; I don't think you'd be going wrong with either.  Both appear to be doing well financially too and should be around for a good while.

P.S. - @BasToTheMax it is on their roadmap but frankly I do not see this as a problem for two reasons:

1. To me, PocketHost is a launch pad not a landing pad - at most maybe you're throwing up some long running MVP's but not a production app.

2.  The beauty of SQLite is it is SO DAMN EASY to maintain / backup / restore - with something like Litestream backing your SQLite implementation you can take it anywhere and not have to worry about such things.  All you'd have to do is restore on deployment but like I said, I don't see this as a final resting place - **Pocketbase hasn't even 1.0'd, let's not forget that and overcomplicate things.**

**Reply by benallfree** [reply]
@pookiepats That's a nice perspective, thanks for sharing it.

I personally plan to run prod projects on PocketHost some day! :) I think it is already a great solution for small and medium sized apps that don't need to scale to millions. Yes, we have a few stability issues but so would anyone hosting it themselves. These just feel more annoying because we often aren't the cause of the downtime we are experiencing so it feels random.

**Reply by nicosql** [reply]
@benallfree yes good point, the perception definitely matters.

I meant no offense, in your docs it says it allows for an "eject" of sorts from the platform so I thought the intention was a launch pad.

So let's say I want to help, in the long term; to help you grow pockethost alongside pocketbase - what technical areas would you suggest I focus on if I wanted to compliment your skill set ?

**Reply by benallfree** [reply]
@pookiepats I didn't take offense, I think it's a great launch pad too :)

If you'd like to contribute, here are some areas where we need help:

* Docs - There are lots of PocketHost features that have sparse or zero documentation.
* Frontend/UX - This is an easy approach to working with the repo and scoring some PRs. It doesn't require much local setup, and just requires knowledge of Svelte
* Backend - For the truly brave, there are many many backend features and ideas being talked about. This requires some pretty deep systems-level expertise as well as deep knowledge of PocketBase, nodejs, Typescript, Linux, and more

Thanks for asking!

**Comment by renatocaliari** [comment]
Thank you for structuring the possible causes and possible initial solutions @benallfree .

Would it help to try running some controlled PocketBase instances by mirroring the infrastructure where PocketHost is, and let them simulate production website patterns (such as queries per minute, execution time, and query complexity) for a few hours and observe how CPU, bandwidth, disk i/o, memory, etc. are maintained and which errors appear in the logs? Maybe this could give more insights?

**Reply by benallfree** [reply]
@renatocaliari I'm sure we might be able to get some information from that. Do you want to give it a try?

**Reply by renatocaliari** [reply]
Unfortunately, I don't have the technical knowledge or skills to help with that task at the moment. 😔

--

*I appreciate all the hard work you have put into developing PocketHost, as well as your willingness to engage with the community by answering questions and trying to resolve issues.

**Comment by benallfree** [comment]
Testing some new features, there may be some instability for a few hours!

**Reply by benallfree** [reply]
Testing has been completed. We will be monitoring for the coming days/weeks to see how stability has improved with v0.8.0!

**Comment by benallfree** [comment]
#234 Reports of expired cert.

@NatumanyaGuy @RobbeDeClerck

Investigating

**Reply by benallfree** [reply]
Please try now. Certbot expired.

**Reply by benallfree** [reply]
<img width="544" alt="image" src="https://github.com/benallfree/pockethost/assets/1068356/e65bd2bc-c812-43fb-9c37-1cae86b9996c">

**Reply by benallfree** [reply]
@renatocaliari

**Reply by renatocaliari** [reply]
Great! Fixed. The platform is working again.

**Reply by RobbeDeClerck** [reply]
Thank you! problem fixed aswell for me here 😄

**Reply by NatumanyaGuy** [reply]
Thank you. Works Now!

**Comment by benallfree** [comment]
PocketHost will be down for scheduled maintenance window while we roll out v0.8.1.

**Reply by benallfree** [reply]
Back up 🎸

**Comment by HEAVYPOLY** [comment]
Are you all running production apps with PH/PB yet?  I love the dx so far but worried about longer term stability.  If I do go with PB/PH is it relatively easy to migrate out to planetscale, supabase etc if needed?

**Reply by benallfree** [reply]
These sound like questions for the PocketBase discussion board since you're asking about PocketBase stability and ease of migration to another platform. PocketHost doesn't get in the way of any of that, but certainly PocketHost wouldn't be a good decision if PocketBase itself isn't a good decision for your projects. Am I understanding your question correctly?

**Reply by HEAVYPOLY** [reply]
Pretty much, wondering if you'd recommend hosting a production app on PH/PB or stick to more hobby / personal projects for now.

**Reply by benallfree** [reply]
If you would run your production app on PocketBase yourself, you'd probably do fine running it on PocketHost as well.

**Reply by HEAVYPOLY** [reply]
Sounds good, thanks

**Comment by znowfox** [comment]
It's down buddy !

**Comment by GroomingMetro** [comment]
looks like it is down

**Comment by benallfree** [comment]
Looking into it.

**Comment by benallfree** [comment]
We are back up, now investigating!

**Comment by benallfree** [comment]
Just to recap, it looks like we had our first outage in a month. This is actually really impressive, because I made a bunch of stability improvements to PocketHost that appear to be very durable. We had a problem with the logging, and I'm not sure if I will be able to determine the cause of this most recent incident, but I did fix the logging issue.

For reference, here's what an outage looks like:

<img width="992" alt="image" src="https://github.com/benallfree/pockethost/assets/1068356/15655160-dd83-4695-9de9-089cab3e425b">

As you can see, bandwidth dropped to nothing and then shortly after that, CPU usage died off as well. To cause this, something would have needed to cause the main `server.ts` daemon to become unresponsive or die. That will remain under investigation. It took a month for it to occur, so I'll wait patiently for the conditions to occur again where I can get more insight as to the cause.

Thanks for the reports @MOBO-Timmy @GroomingMetro

**Comment by benallfree** [comment]
Getting reports of 403 errors from #253 and #254 and #255. Investigating.

**Reply by MalikWhitten67** [reply]
Thanks that was fast lol

**Reply by benallfree** [reply]
Ok, we are back up again. It looks like the v0.16.10 migrations took a LONG time and that caused an outage. Combing through logs now for more details.

**Comment by benallfree** [comment]
#259 reports an outage. Investigating...

**Reply by benallfree** [reply]
We are back up. Looking into the cause now.

**Comment by cheskoxd** [comment]
Hi!

Seems that is currently down... 

https://pockethost.io/ site not showing the Login / Dashboard buttons 
and if you go to https://pockethost.io/dashboard, it redirects you to the homepage

Thanks

**Comment by HEAVYPOLY** [comment]
Same here, seeing login button for a split second then it disappears.

Unexpected response {"url":"","status":0,"data":{},"isAbort":false,"originalError":{"cause":{"errno":-111,"code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":8090}},"name":"ClientResponseError 0"} from mothership

**Comment by benallfree** [comment]
@HEAVYPOLY @cheskoxd Back up, thanks for the report.

Investigating the cause now. Usually it ends up being some corner case that wasn't properly handled in code. Every time we go down, the system gets stronger! Thank you again!

**Comment by HEAVYPOLY** [comment]
Great, thank yoU!

**Comment by benallfree** [comment]
FYI possible dns outages today as we move to cloudflare.

**Reply by niku2676** [reply]
is every api call which is made in the meantime deleted?

**Reply by benallfree** [reply]
I’m not sure what you mean by deleted. Some API calls may fail to connect during this period.

**Reply by niku2676** [reply]
yeah thats why I mean. that is not good

**Reply by benallfree** [reply]
any backend outage for an application means API calls will not be successful. Yes, outages are not good . 🚫🙈

do you have a special case I am not understanding?

**Comment by renatocaliari** [comment]
About 10 minutes ago, I noticed that my website has been experiencing errors due to the connection with PocketBase timing out repeatedly.

**Reply by renatocaliari** [reply]
It wasn't something temporary. It continues to be unstable and giving errors in the connections with the database.

**Reply by benallfree** [reply]
@renatocaliari Please check now and see if things have stabilized.

**Reply by renatocaliari** [reply]
Tks. Yes, it seems like things are back on track. Any insights into what might have caused the issue?

**Reply by benallfree** [reply]
@renatocaliari I'm reviewing the logs now. I'll update this thread when I have more info.

**Reply by benallfree** [reply]
@renatocaliari which subdomain was having issues?

**Reply by renatocaliari** [reply]
Do you mean the name of my app / pocketbase instance inside PocketHost?

**Reply by benallfree** [reply]
Yes please, that way I can check specifically for errors on your instance.

**Reply by renatocaliari** [reply]
`chubby-zoo`

**Reply by benallfree** [reply]
I see these two errors in the log:

```bash
[2023-09-03T11:50:33.569Z] [ERROR] [server.ts] [InstanceService] [chubby-zoo::~0.16.5] [pid:4184389] Error: Timeout waiting for pid to die
```

It seems possible that your instance became unresponsive. Are you running any long queries or dealing with complex data?

**Reply by renatocaliari** [reply]
They are simple queries. The database is very small. And it started giving an error without me changing anything.

**Reply by benallfree** [reply]
Ok it's unlikely you caused it. I will keep investigating. I am tracking a few intermittent errors that cause overall instability, so it's possible something else was going on and you just happened to be the one to notice it first.

**Comment by FrenchFryInteractive** [comment]
My instance seems to be down. Site can't be reached.

**Reply by benallfree** [reply]
How about now?

**Reply by FrenchFryInteractive** [reply]
Seems fixed. What was wrong?

**Reply by benallfree** [reply]
@FrenchFryInteractive Still investigating. The outages generally have to do with race conditions that appear only in production under very specific conditions, but I'll let you know what I find :)

**Comment by RA9** [comment]
My instance is giving me the below error

![image](https://github.com/benallfree/pockethost/assets/20514920/0a41c362-3e23-4973-8193-0ff1b642949e)

**Reply by RA9** [reply]
Now it's just in the redirect loop

[Screencast from 2023-09-15 12-04-17.webm](https://github.com/benallfree/pockethost/assets/20514920/0195a2ef-3bb6-4ea7-9279-774b36cf1a6f)

**Reply by benallfree** [reply]
Investigating...

**Reply by benallfree** [reply]
Should be back up, I'm still looking into the underlying cause.

**Reply by DiceBreakers** [reply]
I am (along with [HarithOmari](https://github.com/HarithOmari)) experiencing problems similar to RA9. TImeout obtaining instance API errors, and occasionally get into the admin dashboard and then "Something went wrong while processing your request" if I take any action.

Instance is starting and stopping shortly after. Tried creating a test database and it's experiencing the same. Original DB is version ~0.17.7, new DB is ~0.18.6.

2023-09-19T15:57:01.806Z info Shutting down instance
2023-09-19T15:56:56.854Z info 
2023-09-19T15:56:56.827Z info └─ Admin UI: http://0.0.0.0:8090/_/
2023-09-19T15:56:56.766Z info ├─ REST API: http://0.0.0.0:8090/api/
2023-09-19T15:56:56.739Z info 2023/09/19 15:56:56 Server started at http://0.0.0.0:8090
2023-09-19T15:56:55.189Z info Starting instance
2023-09-19T15:55:18.872Z info Shutting down instance
2023-09-19T15:55:12.864Z info 
2023-09-19T15:55:12.579Z info └─ Admin UI: http://0.0.0.0:8090/_/
2023-09-19T15:55:12.446Z info ├─ REST API: http://0.0.0.0:8090/api/
2023-09-19T15:55:12.417Z info 2023/09/19 15:55:12 Server started at http://0.0.0.0:8090
2023-09-19T15:55:10.459Z info Starting instance

**Reply by burraksumer** [reply]
I have almost the same issue where I can't access the admin panel for some reason and I get timeouts or 403s to api requests.

**Reply by argonptg** [reply]
I've been having the same issue where it goes to idle, i also don't have any hooks on PB

**Reply by benallfree** [reply]
@DiceBreakers @burraksumer @Brisolo32 PocketHost instances going idle is normal and is not related to any outage. PocketHost hibernates instances when there are no active HTTP connections.

**Comment by HEAVYPOLY** [comment]
Getting this on dashboard:  Unexpected response {"url":"","status":0,"data":{},"isAbort":false,"originalError":{"cause":{"errno":-111,"code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":8090}},"name":"ClientResponseError 0"} from mothership

**Comment by twiddlecoding** [comment]
I get this Unexpected response {"url":"","status":0,"data":{},"isAbort":false,"originalError":{"cause":{"errno":-111,"code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":8090}},"name":"ClientResponseError 0"} from mothership

And my service is down. Please fix this immediately.

**Reply by twiddlecoding** [reply]
It seems to be working now, what happened?

**Comment by benallfree** [comment]
@twiddlecoding @HEAVYPOLY @burraksumer @DiceBreakers Please try now. As per usual, investigating the cause takes some time but restarting the system is easy and effective :)

I'll update this thread more when I find out what is failing. I soft-released 0.9.0 which Dockerizes things, so there could be some instability with that. Typically the issues revolve around processing failing and the main thread giving up on serving more. Always looking for ways to continue improving this.

**Reply by DiceBreakers** [reply]
Thank you very much! I'm back up and running.

I appreciate you running the service and managing all the moving parts.

**Comment by benallfree** [comment]
FYI there may be some instability this morning as I roll out some enhancements. Should't be too noticeable though.

**Reply by twiddlecoding** [reply]
Everything was down for a couple of minutes just now. Seems to be okay now.

**Reply by benallfree** [reply]
@twiddlecoding It will continue for an hour or so. Database upgrade!

**Reply by twiddlecoding** [reply]
Will you keep doing things like this? That stops the servers. I'm running a saas (no profits yet, but 400 users) on Pockethost. Excpecting to pay when you start to charge, but did not expect the server to go down every now and then.

**Reply by benallfree** [reply]
@twiddlecoding 400 users is amazing!! Congratulations on that accomplishment. Right now PocketHost is an as-is service. If we moved to a paid tier, we'd need to address these types of issues more carefully. I think, due to the nature of PocketBase itself, some downtime is inevitable. But we can reduce it to seconds instead of minutes.

**Reply by tenderassist** [reply]
Is there any possibility that enhancements can be made during days with low traffic for example a Sunday? I use your service for business purposes and it interferes with our tasks when the servers go down every now and then. Other than that I love pockethost, good work!

**Reply by benallfree** [reply]
@tenderassist Yes, I think that makes sense as the service grows and people begin relying upon it for production needs. 

There are a lot of things I can do to optimize for uptime. Based on feedback and knowing how quickly people notice when the service is down, it sounds like this should become a point of focus :)

**Reply by benallfree** [reply]
PS - one thing I can do is keep a staging server so I understand the complexity of the upgrade. For example, today's upgrade is doing a database migration to the millions of invocation records stored in the central server. It seems quite long.

**Reply by benallfree** [reply]
Ok, we should be back to stability now. Please let me know if anything seems amiss.

**Reply by twiddlecoding** [reply]
It seems like your service is down at the moment

**Reply by benallfree** [reply]
@twiddlecoding Thank you, it looks like at least some services were running but I restarted everything just to be sure. We seem to have an issue that occurs every 2-4 days, still investigating :)

**Comment by twiddlecoding** [comment]
Your app is down

**Reply by benallfree** [reply]
We are back up. If anyone has time to send PRs are help troubleshoot, we are over in https://discord.com/channels/1128192380500193370/1159322063539023922 working on it.

**Comment by gideongrinberg** [comment]
The new update appears to have caused an outage.

**Reply by benallfree** [reply]
Definitely instability with the changeover to Docker at 0.9.0. Expected and necessary, but very annoying because things were pretty smooth before. We have all the logs and will continue investigating to track down the root causes of these outages.

**Comment by benallfree** [comment]
Everyone, please join us on Discord https://discord.gg/nVTxCMEcGT

This thread has moved to https://discord.com/channels/1128192380500193370/1179852349011939439

Discord provides a much better forum for troubleshooting issues in real time.

**Comment by benallfree** [comment]
By popular vote, this has been reopened and is now the primary support thread.

**Comment by mmgfrcs** [comment]
Note that this remains true imo:
> Discord provides a much better forum for troubleshooting issues **in real time**.

Anyway, for some reason file uploads are sooo slow today

**Comment by hexiotek** [comment]
is it down? i have 4 pocketbases and i cant access 3 of them, only one pops up on my dashboard ""code":400,"message":"Something went wrong while processing your request."

**Reply by allendesignanddev** [reply]
Our instance is also down with a 500 error

**Reply by benallfree** [reply]
@hexiotek @allendesignanddev Please try now. When the frontend (pockethost.io web site) can't access the mothership, it doesn't surface the error and instead just displays a blank list. That scares everyone into thinking their instances have vanished. I'll take steps to make a better error message.

@allendesignanddev When you say your instance is down with a 500 error, please provide the exact URL you mean. I can't tell if you mean the PocketHost dashboard, or your actual PocketBase instance URL.

**Reply by allendesignanddev** [reply]
@benallfree its working now. Thanks! I'll make sure to provide the url next time.

**Reply by benallfree** [reply]
I've updated the PocketHost dashboard to be clearer about when the connection to the mothership is lost vs an actual outage.

**Reply by Dylan70110** [reply]
Bonjour gros soucis j'arrive plus à accéder à mon serveur depuis 23h hier soir, je peut plus aller sur le panel admin pour lancer le serveur pouvez vous m'aider je suis avec pockethost

**Reply by benallfree** [reply]
@Dylan70110 Bonjour, je ne parle pas français, mais ChatGPT le fait. Veuillez rejoindre notre canal Discord où je pourrai vous aider davantage, ou peut-être qu'un membre francophone de la communauté pourra le faire. Consultez https://status.pockethost.io, tout fonctionne, donc il est probable que le problème soit lié à votre compte ou à votre instance PocketBase. 😎

**Reply by Dylan70110** [reply]
Le lien discord net plus valide

**Reply by benallfree** [reply]
@Dylan70110 https://pockethost.io/support and https://discord.gg/nVTxCMEcGT works?
